[TR] Guides (#3638)
ppy/osu-wiki: wiki/Guides/tr.md

# Kılavuzlar

> aşağıda osu!community üyeleri tarafından yaratılmış; pek çoğu osu!forumlarda başlayıp buraya taşınmış olan kılavuzların sıralı listesi mevcuttur.

Reviewer: ```suggestion
Aşağıda osu!community üyeleri tarafından yaratılmış; pek çoğu osu!forumlarda başlayıp osu! wiki'ye taşınmış olan kılavuzların sıralı listesi mevcuttur.
```

## Beatmap oluşturma

*Ana sayfa: [Beatmap oluşturma](/wiki/Beatmapping)*

- [Özel Vuruş Sesleri Ekleme](Adding_Custom_Hitsounds)
- [Ses Düzenleme](Audio_Editing)
- [Sanatçı ya da Başlık Adı Değiştirme](Changing_the_Artist_or_Title)

> - [Başlık Metni Değiştirme](/wiki/Beatmaps/Title_Text#changing-title-text)

Reviewer: ```suggestion
- [Başlık Metnini Değiştirme](/wiki/Beatmaps/Title_Text#changing-title-text)
```

> - [Collab Bilgilendirme](Collab_Information)

Reviewer: ```suggestion
- [Collab Bilgisi](Collab_Information)
```

- [Dosyaları Sıkıştırmak](Compressing_Files)
- [Özel Vuruş Sesi Kütüphanesi](Custom_Hitsound_Library)
- [Özel Ses Seti Bindirmeleri](Custom_Sample_Overrides)
- [Video Oyunlarından Şarkıları Almak](Getting_Songs_From_Video_Games)
- [Şarkılar Naısl Zamanlanır](How_to_Time_Songs)
- [Müzik Teorisi](Music_Theory)
- [osu!mania Map Yapım Kılavuzu](osu!mania_Mapping_Guide)
- [Ofseti Doğru Vuruşa Ayarlamak](Setting_the_Offset_on_the_Correct_Beat)
- [Bir Beatmap Projesi Başlatmak](Starting_a_Beatmap_Project)
- [YouTube'dan Videolar](Videos_From_Youtube)

## Modlama

*Ana sayfa: [Modlama](/wiki/Modding)*

- [Mapinizi Modlatmak](Getting_Your_Map_Modded)

## Oynama

- [Yeni Başlayanlar İçin Öğretici](Beginner's_Tutorial)
- [osu!mania Nasıl Oynanır](How_to_Play_osu!mania)
- [Ofset Sihirbazı Nasıl Kullanılır](How_to_Use_the_Offset_Wizard)
- [Beatmaplari Aramak ve İndirmek](Searching_and_Downloading_Beatmaps)
- [Tablet Satın Almak](Tablet_Purchase)

> - [Yetenek Geliştirmek Hakkında İpuçları](Tips_and_Tricks_on_Skill_Improvement)

Reviewer: ```suggestion
- [Yetenek Geliştirme Üzerine İpuçları](Tips_and_Tricks_on_Skill_Improvement)
```

## Tema yapımı

> *Ana sayfa: [Tema Yapımı](/wiki/Skinning)*

Reviewer: ```suggestion
*Ana sayfa: [Tema yapımı](/wiki/Skinning)*
```

> *Ayrıca bakınız: [Tema Yapımı Kılavuzu](/wiki/Skinning_Tutorial)*

Reviewer: ```suggestion
*Ayrıca bakınız: [Tema Yapma Öğreticisi](/wiki/Skinning_Tutorial)*
```

> - [Karışık Arkaplanlar Kırpmak](Cropping_with_Complex_Backgrounds)
> - [Basit Arkaplanlar Kırpmak](Cropping_with_Simple_Backgrounds)

Reviewer: ```suggestion
- [Karışık Arkaplanları Kırpmak](Cropping_with_Complex_Backgrounds)
- [Basit Arkaplanları Kırpmak](Cropping_with_Simple_Backgrounds)
```

- [Düzgünce Ortalanmış Spinnerlar Yapmak](Making_Properly_Centered_Spinners)

## Diğer

> - [Discord Rich Presence](Discord_Rich_Presence)

Reviewer: ```suggestion
- [Discord Zengin İçerik](Discord_Rich_Presence)
```
let's use the Turkish term for it

Author: I'm going to leave it as is for now as the official Turkish web page of Discord didn't bother translating it.
